processing/move_file_derivatives.py

```diff
@@ -0,0 +1,40 @@
+import os
```

Comment:
can you please add a header to briefly explain what this function does

Reply:
Done in 48edbd7